examples/storage/cassandra/README.md

```diff
@@ -176,25 +177,277 @@ spec:
 [Download example](cassandra-service.yaml?raw=true)
 <!-- END MUNGE: EXAMPLE cassandra-service.yaml -->
 
-An important thing to note here is the `selector`. It is a query over labels,
-that identifies the set of Pods contained by this Service.  In this case the
-selector is `app=cassandra`. If there are any pods with that label, they will be
-selected for membership in this service. We'll see that in action shortly.
+Create the service for the Pet Set:
 
-Create the Cassandra service as follows:
 
 ```console
 $ kubectl create -f examples/storage/cassandra/cassandra-service.yaml
 ```
 
+The following command shows if the service has been created.
+
+```console
+$ kubectl get svc cassandra
+```
+
+The response should be like:
+
+```console
+NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+cassandra   None         <none>        9042/TCP   45s
+```
+
+If an error is returned the service create failed.
+
+## Step 2: Use a Pet Set to create Cassandra Ring
+
+Pet Sets are a new feature that was added as an <i>Alpha</i> component in Kubernetes
+1.3.  Deploying stateful distributed applications, like Cassandra, within a clustered
+environment can be challenging.  We implemented Pet Set to greatly simplify this
+process.  Multiple Pet Set features are used within this example, but is out of
+scope of this documentation.  [Please refer to the Pet Set documentation.](../../../docs/user-guide/petsets.md)
```

Comment:
Same here, invalid path. Can point to http://kubernetes.io/docs/user-guide/petset/ ?